Board.java

```diff
@@ -0,0 +1,634 @@
+package test;
+import java.util.ArrayList;
+
+public class Board {
+    public class BoardSquare {
+        private Tile tile;
+        private boolean isOccupied;
+        private int letterMultiplier;
+        private int wordMultiplier;
+
+        private BoardSquare(Tile tile, boolean isOccupied, int letterMultiplier, int wordMultiplier) {
+            this.tile = tile;
+            this.isOccupied = isOccupied;
+            this.letterMultiplier = letterMultiplier;
+            this.wordMultiplier = wordMultiplier;
+        }
+
+        private BoardSquare() {
+            this(null, false, 1, 1);
+        }
+
+        private BoardSquare(int letterMultiplier, int wordMultiplier) {
+            this(null, false, letterMultiplier, wordMultiplier);
+        }
+
+        public Tile getTile() {
+            return tile;
+        }
+
+        public void setTile(Tile tile) {
+            this.tile = tile;
+            this.isOccupied = true;
+        }
+
+        public boolean isOccupied() {
+            return isOccupied;
+        }
+
+        public void setOccupied(boolean occupied) {
+            isOccupied = occupied;
+        }
+
+        public int getLetterMultiplier() {
+            return letterMultiplier;
+        }
+
+        public void setLetterMultiplier(int letterMultiplier) {
+            this.letterMultiplier = letterMultiplier;
+        }
+
+        public int getWordMultiplier() {
+            return wordMultiplier;
+        }
+
+        public void setWordMultiplier(int wordMultiplier) {
+            this.wordMultiplier = wordMultiplier;
+        }
+    }
+
+    private final int BOARD_SIZE = 15;
+    private BoardSquare[][] board = new BoardSquare[][]{{
```

Comment:
how can you make this shorter?

Reply:
do you mean like that:
private BoardSquare[][] board = {{